content/news/posts/2024/do-you-speak-prestashop.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "PrestaShop DevCon 2023: Do you speak PrestaShop"
+subtitle: "How the community works to translate the software into multiple languages."
+date:   2024-02-05
+authors: [ JulieVarisellaz, NicolasLoeuillet ]
+icon: icon-leaf
+tags: [ news, event, community ]
+---
+
+## Do you speak PrestaShop?
+
+On November 30, Nicolas Lœuillet and Julie Varisellaz gave a presentation about the project’s translations at the first-ever PrestaShop Developer Conference.
```

Comment:
```suggestion
On November 30th, Nicolas Lœuillet and Julie Varisellaz gave a presentation about the project’s translations at the first-ever PrestaShop Developer Conference.
```

Reply:
"On November 30" is correct. See Grammarly's article: 
<img width="738" alt="Screenshot 2024-02-06 at 14 22 57" src="https://github.com/PrestaShop/prestashop.github.io/assets/70583503/cf791eef-acd8-4193-8f87-8f85348757a5">
https://www.grammarly.com/blog/how-to-write-dates/#:~:text=Many%20people%20get%20confused%20about,13th%2C%202007%20Daniel%20was%20born.